lib/jsonapi_parameters/validator.rb

```diff
@@ -0,0 +1,52 @@
+require 'active_model'
+require 'json_schemer'
+
+module JsonApi::Parameters
+  SCHEMA_PATH = 'spec/support/jsonapi_schema.json'.freeze
```

Comment:
why is the production-use schema file put in specs?

Reply:
Good point. Where would be a good spot to place it? Root directory? lib?

Reply:
I vote for lib. Root directory is a sacred place :)